config/default/user.role.organisation_admin.yml

```diff
@@ -1,44 +0,0 @@
-uuid: 91817ee7-8531-437a-9538-b91f274e4312
-langcode: en
-status: true
-dependencies:
-  config:
-    - node.type.student
-  module:
-    - block
-    - company
```

Comment:
@archanarkamath 
Since you are removing "company" module. and compnay module has been mentioned here. it is linked each other.
So when you uninstalled it, this organisation_admin role removed config shows you.

what you could have done is ....

1. Discard all the config files related organisation_admin role.  not supposed to be commit.
2. here where - Comapny is mentioned you could have changed it to "organisation" manually. OR removed it completely manually.
3. And then commit only this config file.